dev-lang/ocaml/ocaml-4.09.0.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+HOMEPAGE="https://ocaml.org/"
+SRC_URI="https://github.com/ocaml/ocaml/archive/${PV}.tar.gz -> ${P}.tar.gz"
+DESCRIPTION="Programming language supporting functional, imperative & object-oriented styles"
+
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+# X flag required by dev-ml/camlimages and others
+IUSE="emacs flambda latex ocamlopt spacetime xemacs X"
+
+RDEPEND="sys-libs/binutils-libs:=
+	spacetime? ( sys-libs/libunwind:= )"
+BDEPEND="${RDEPEND}
+	virtual/pkgconfig"
+PDEPEND="emacs? ( app-emacs/ocaml-mode )
+	xemacs? ( app-xemacs/ocaml )"
+
+src_configure() {
+	local opt=(
+		--bindir="${EPREFIX}/usr/bin"
+		--libdir="${EPREFIX}/usr/$(get_libdir)/ocaml"
+		--mandir="${EPREFIX}/usr/share/man"
+		--prefix="${EPREFIX}/usr"
+		$(use_enable flambda)
+		$(use_enable spacetime)
+	)
+	# Upstream build ignores LDFLAGS in several places.
```

Comment:
Also, please move the whole sed calls to the `src_prepare()`

Reply:
Will fix.